ui/src/app/intro/views/Intro.tsx

```diff
@@ -1,14 +1,35 @@
+import type { ReactNode } from "react";
+
+import { Spinner } from "@canonical/react-components";
+import { useSelector } from "react-redux";
 import { Route, Switch } from "react-router-dom";
 
 import ImagesIntro from "./ImagesIntro";
+import IncompleteCard from "./IncompleteCard";
 import MaasIntro from "./MaasIntro";
 import MaasIntroSuccess from "./MaasIntroSuccess";
 import UserIntro from "./UserIntro";
 
+import Section from "app/base/components/Section";
 import NotFound from "app/base/views/NotFound";
 import introURLs from "app/intro/urls";
+import authSelectors from "app/store/auth/selectors";
 
 const Intro = (): JSX.Element => {
+  const loading = useSelector(authSelectors.loading);
+  const authUser = useSelector(authSelectors.get);
+  let content: ReactNode;
+  if (loading) {
+    content = <Spinner text="Loading..." />;
+  } else if (authUser && !authUser.is_superuser) {
```

Comment:
Shouldn't this also check that `config.completed_intro === false`?

Reply:
Yes, thanks. That change currently means that the user can visit the intro after the maas has been configured, but that should get resolved once we [add the redirects](https://github.com/canonical-web-and-design/app-squad/issues/159).